test/v1/connectors/PensionChargesConnectorSpec.scala

```diff
@@ -57,67 +58,152 @@ class PensionChargesConnectorSpec extends ConnectorSpec {
+            )
+            .returns(Future.successful(expected))
+
+          await(connector.deletePensionCharges(deleteRequest)) shouldBe expected
+        }
+      }
+
+      "a request returning multiple errors" should {
+        "return an unsuccessful response with the correct correlationId and multiple errors" in new IfsTest with Test {
+
+          def taxYear: TaxYear = TaxYear.fromMtd("2019-20")
+
+          val expected = Left(ResponseWrapper(correlationId, Seq(NinoFormatError, StandardDownstreamError)))
 
-        await(connector.deletePensionCharges(deleteRequest)) shouldBe expected
+          MockedHttpClient
+            .delete(
+              url = s"$baseUrl/income-tax/charges/pensions/$nino/${taxYear.asMtd}",
+              config = dummyIfsHeaderCarrierConfig,
+              requiredHeaders = requiredIfsHeaders,
+              excludedHeaders = Seq("AnotherHeader" -> "HeaderValue")
+            )
+            .returns(Future.successful(expected))
+
+          await(connector.deletePensionCharges(deleteRequest)) shouldBe expected
+        }
       }
     }
 
-    "a request returning multiple errors" should {
-      "return an unsuccessful response with the correct correlationId and multiple errors" in new IfsTest with Test {
-        def taxYear: TaxYear = TaxYear.fromMtd("2019-20")
+    "Amend pension charges" when {
 
-        val expected = Left(ResponseWrapper(correlationId, Seq(NinoFormatError, StandardDownstreamError)))
+      "a valid request is supplied" should {
+        "return a successful response with the correct correlationId" in new IfsTest with Test {
+          def taxYear: TaxYear = TaxYear.fromMtd("2019-20")
 
-        MockedHttpClient
-          .delete(
-            url = s"$baseUrl/income-tax/charges/pensions/$nino/${taxYear.asMtd}",
-            config = dummyIfsHeaderCarrierConfig,
-            requiredHeaders = requiredIfsHeaders,
-            excludedHeaders = Seq("AnotherHeader" -> "HeaderValue")
-          )
-          .returns(Future.successful(expected))
+          val expected = Right(ResponseWrapper(correlationId, Unit))
+
+          MockedHttpClient
+            .put(
+              url = s"$baseUrl/income-tax/charges/pensions/$nino/${taxYear.asMtd}",
+              body = pensionCharges,
+              config = dummyIfsHeaderCarrierConfig,
+              requiredHeaders = requiredIfsHeaders,
+              excludedHeaders = Seq("AnotherHeader" -> "HeaderValue")
+            )
+            .returns(Future.successful(expected))
+
+          await(connector.amendPensionCharges(amendRequest)) shouldBe expected
+        }
+      }
+
+      "a valid request is supplied for a Tax Year Specific tax year" should {
+        "return a successful response with the correct correlationId" in new TysIfsTest with Test {
+          def taxYear: TaxYear = TaxYear.fromMtd("2023-24")
+
+          val expected = Right(ResponseWrapper(correlationId, retrieveResponse))
```

Comment:
why is this testing for the retrieveResponse? for create and amend we would expect an empty response (`Unit`)